src/test/java/de/tum/in/www1/artemis/AbstractSpringIntegrationLocalCILocalVCTest.java

```diff
@@ -59,7 +59,8 @@
         "artemis.version-control.local-vcs-repo-path=${java.io.tmpdir}", "artemis.build-logs-path=${java.io.tmpdir}/build-logs",
         "artemis.continuous-integration.specify-concurrent-builds=true", "artemis.continuous-integration.concurrent-build-size=1",
         "artemis.continuous-integration.asynchronous=false", "artemis.continuous-integration.build.images.java.default=dummy-docker-image",
-        "artemis.continuous-integration.image-cleanup.enabled=true", "spring.liquibase.enabled=true" })
+        "artemis.continuous-integration.image-cleanup.enabled=true", "artemis.continuous-integration.image-cleanup.disk-space-threshold-mb=1000000000",
```

Comment:
Setting `artemis.continuous-integration.image-cleanup.disk-space-threshold-mb` is meant to clean up all unused images? Why don't we set it to a more realistic value, e.g. the 2000 Mb that are actually used in the production code?

Reply:
Explanation in testing session: we can't mock the usable space with Mockito -> we set it to a very high value to make sure it gets cleaned up